### ISSUE TITLE
Add code linter and it's GitHub action

### ISSUE THREAD

**Issue by vj-codes** [techgaun/github-dorks#53]
To maintain consistency of code, add a code linter like black and it's GitHub action to check if it's properly formatted when a pull request is created

**Comment by vj-codes** [comment]
@techgaun hey what do you think about this issue? 
I would like to work on it if approved:)

**Comment by techgaun** [comment]
yup @vj-codes this would be a great addition for sure. you can have it.

**Comment by harshalchaudhari35** [comment]
@techgaun is this still open? As I don't see much traction, I can pick this is up if still available.

**Comment by vj-codes** [comment]
@harshalchaudhari35 sorry I'm interested to work on it, will create a PR soon:)

**Comment by harshalchaudhari35** [comment]
@vj-codes ok good luck
